saos-webapp/src/main/java/pl/edu/icm/saos/webapp/judgment/detail/JudgmentDetailsSortService.java

```diff
@@ -0,0 +1,80 @@
+package pl.edu.icm.saos.webapp.judgment.detail;
+
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.stereotype.Service;
+
+import pl.edu.icm.saos.persistence.model.Judge;
+import pl.edu.icm.saos.persistence.model.Judge.JudgeRole;
+import pl.edu.icm.saos.persistence.model.Judgment;
+
+/**
+ * Service for sorting judges in judgment. 
+ * 
+ * @author Łukasz Pawełczak
+ *
+ */
+@Service
+public class JudgmentDetailsSortService {
+
+	
+	//------------------------ LOGIC --------------------------
+	
+	/**
+	 * Sort judges {@link pl.edu.icm.saos.persistence.model.Judge} in judgment {@link pl.edu.icm.saos.persistence.model.Judgment}.
+	 * 
+	 * @param Judgment
+	 * @return Judgment with sorted Judges 
+	 */
+	public Judgment sortJudges(Judgment judgment) {
+		
+		List<Judge> judges  = judgment.getJudges().stream().collect(Collectors.toList());
```

Comment:
`judgment.getJudges().stream().collect(Collectors.toList())` could be replaced by simple `judgment.getJudges()`